Giving choice - npm or yarn.

NTillmann opened the pull request:
Fixing issue raised in #782.

cblappert commented on README.md:
````diff
@@ -7,10 +7,10 @@ See the official [prepack.io](http://prepack.io) website for an introduction and
 
 ```bash
-$ yarn global add prepack
+$ npm install -g yarn
````
This installs yarn not prepack, `npm install -g prepack` installs prepack

NTillmann commented:
Updated README. Using npm for _using_ Prepack should be fine. You only need all the scripts for further development work. I added a note describing that.

cblappert commented:
In this case we should also change the `package.json` to replace `npm` in all the scripts with `yarn` (e.g. `validate`, `prepublish`).

NTillmann commented:
Okay, you one really needs yarn to develop.
